dist/redhat/scylla.spec

```diff
@@ -128,7 +137,8 @@ ln -sfT /etc/scylla /var/lib/scylla/conf
 %{_bindir}/iotune
 %{_bindir}/scyllatop
 %{_bindir}/nodetool
-%{_sbindir}/scylla*
+%{_sbindir}/scylla_*setup
+%{_sbindir}/scylla_kernel_check
 %{_sbindir}/node_health_check
 %{_sbindir}/seastar-cpu-map.sh
```

Comment:
Why not install unconditionally into _bindir? There's no real reason to use /usr/sbin.

Reply:
i wanted to preserve the existing behavior. what if user has some tools which hardcode the path to `/usr/sbin/scylla_kernel_check` ?

Reply:
but if this does not matter, i am happy to install these executables into `/usr/bin`. it is a lot simpler.

Reply:
one of the "toolings" i found: https://src.fedoraproject.org/rpms/systemd/c/a37923658fbe9f511c36d31f556eaada782691eb?branch=rawhide

Reply:
i will go with `filesystem(unmerged-sbin-symlinks)`. cooking the compatible change.

Reply:
@avikivity unfortunately, our own test is calling `scylla_setup` with its full path. see https://github.com/scylladb/scylla-cluster-tests/blob/3787115a6a9e023266a168a2a44201719c859f87/sdcm/cluster.py#L4686 . so i reverted to the original version.